plv8.cc

```diff
@@ -2019,6 +2115,7 @@ Datum
 Converter::ToDatum(Handle<v8::Value> value, Tuplestorestate *tupstore)
 {
 	Isolate		   *isolate = Isolate::GetCurrent();
+    Local<Context>  context = isolate->GetCurrentContext();
```

Comment:
and all of the hundreds of places where local context is used gratuitously, which is rather frustrating.